.gitignore

```diff
@@ -53,4 +53,8 @@ AOE-exp/
 !/TGC-HFM-Style-Culture-Colors/
 !TGC-HFM-Style-Culture-Colors.mod
 !/TGC-Statehood-Mechanic/
-!TGC-Statehood-Mechanic.mod
\ No newline at end of file
+!TGC-Statehood-Mechanic.mod
+TGC/map/rivers.bmp
```

Comment:
why

Reply:
> Apart from the .gitignore edit (which I assume is an accident, idk what river.bmp has with political parties), this looks okay - revert that file and feel free to merge.

Yeah that's an accident (Both gitignore and rivers lol) I was editing the rivers.bmp file to test how the river editing thing worked, but I didn't want the file to be included in the PR so I tried to make GitHub ignore the file (I thought it ignored the rivers.bmp so I didn't pay much attention to the PR when making it)

I guess that's how I messed up and ended up with both the Rivers.bmp and the gitignore files into the PR.
Give me a moment and I'll fix it